src/distributors/zulu-installer.ts

```diff
@@ -154,4 +152,13 @@ export class ZuluDistributor extends JavaBase {
         return process.platform;
     }
   }
+
+  private getDownloadArchiveExtension() {
```

Comment:
probably it makes sense to move it to util or base-installer since it is used for both Adopt and Zulu implemetation and most likely will be used for other providers.

Reply:
Done. Thank you.